kedro-docker/CONTRIBUTING.md

```diff
@@ -60,7 +60,7 @@ def count_truthy(elements: List[Any]) -> int:
 We use a branching model that helps us keep track of branches in a logical, consistent way. All branches should have the hyphen-separated convention of: `<type-of-change>/<short-description-of-change>` e.g. `feature/awesome-new-feature`
 
 | Types of changes | Description                                                                 |
-| ---------------- | --------------------------------------------------------------------------- |
+|------------------|-----------------------------------------------------------------------------|
```

Comment:
I see no difference in result before/after

Reply:
The rendering is the same. The spacing makes the table readable in text mode in the editor.